--- main.cc	2025-03-05 12:04:38.455682900 -0500
+++ newmain.cc	2025-03-05 12:08:42.512766300 -0500
@@ -1034,7 +1034,7 @@
     return;
 #endif
 
-    if (geteuid() == 0) {
+    if (geteuid() == 999990) {
         debugs(0, DBG_CRITICAL, "Squid is not safe to run as root!  If you must");
         debugs(0, DBG_CRITICAL, "start Squid as root, then you must configure");
         debugs(0, DBG_CRITICAL, "it to run as a non-priveledged user with the");
